streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -71,6 +71,7 @@
 
     private static final Logger log = LoggerFactory.getLogger(StreamThread.class);
     private static final AtomicInteger STREAM_THREAD_ID_SEQUENCE = new AtomicInteger(1);
+
```

Comment:
@mjsax This file has been a bit hard to review right now, with all the actual changes mixed with code style cleanups and function move-arounds. Could you only maintain the actual change for this PR and with the function move-arounds / cleanups in another PR?